### PR TITLE
feat(loader): add loader component

### PR DESCRIPTION
**Pull request by celinekurpershoek**
New loader component

### CODE REVIEW

File: projects/web-app/src/components/app-shell/app-shell.vue [hunk 1]
```diff
@@ -16,18 +18,35 @@
 </template>
 
 <script>
+import { ref, onUpdated, onErrorCaptured } from 'vue';
 import AppHeader from './app-header';
 import AppSidebar from './app-sidebar';
+import Loader from '../loader/loader';
```
Comment (celinekurpershoek) [inline projects/web-app/src/components/app-shell/app-shell.vue:24]: Should this component become a consumable?

Comment (vvandijck) [reply projects/web-app/src/components/app-shell/app-shell.vue:24]: @bob-voorhoede Can it and should it be a composable?

File: projects/web-app/src/components/app-shell/app-shell.vue [hunk 2]
```diff
@@ -16,18 +18,35 @@
...
+    return {
+      headData,
+      loading,
+    };
+  },
 
   components: {
     AppHeader,
     AppSidebar,
+    Loader
```
Comment (vvandijck) [inline projects/web-app/src/components/app-shell/app-shell.vue]: Add missing `,`

File: projects/web-app/src/components/loader/loader.vue [hunk 3]
```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="loader-container">
+    <div class="loader"></div>
+  </div>
+</template>
+
+<style scoped>
```
Comment (vvandijck) [inline projects/web-app/src/components/loader/loader.vue]: `<style>` should be fine

File: projects/web-app/src/components/loader/loader.vue [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+.loader-container {
+  display: grid;
+  align-content: center;
+  height: 100%;
+}
```
Comment (vvandijck) [inline projects/web-app/src/components/loader/loader.vue:12]: Add whitespace

File: projects/web-app/src/components/loader/loader.vue [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+.loader {
+	width: 30px;
+	height: 30px;
+	position: relative;
+  margin: auto;
+
+}
```
Comment (vvandijck) [inline projects/web-app/src/components/loader/loader.vue:19]: Same

File: projects/web-app/src/components/loader/loader.vue [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+.loader::before, .loader::after {
+		content: '';
+		position: absolute;
+}
```
Comment (vvandijck) [inline projects/web-app/src/components/loader/loader.vue:24]: Same

File: projects/web-app/src/components/loader/loader.vue [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+.loader-list {
+		display: grid;
+		grid-template: repeat(3, 120px) / repeat(3, 120px);
+		grid-gap: 30px;
+}
```
Comment (vvandijck) [inline projects/web-app/src/components/loader/loader.vue:30]: Same

File: projects/web-app/src/components/loader/loader.vue [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+.loader-item {
+		display: flex;
+    background: rgba(0, 0, 0, 0.05);
+}
+
+.loader {
+	border: 1px solid blue;
+	height: 14px;
+  width: 32px;
+}
```
Comment (vvandijck) [inline projects/web-app/src/components/loader/loader.vue:41]: Same

File: projects/web-app/src/components/loader/loader.vue [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+.loader::before {
+  width: 4px;
+  height: 10px;
+  background: blue;
```
Comment (vvandijck) [inline projects/web-app/src/components/loader/loader.vue]: `background-color` is more specific.

**Review by vvandijck — CHANGES_REQUESTED** [review]

**Review by vvandijck — APPROVED** [review]

**Pull request merged**